modules/apps/message-boards/message-boards-web/src/main/resources/META-INF/resources/message_boards_admin/js/BanUsersManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {postForm} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {banUsersURL},
+	portletNamespace,
+	...otherProps
+}) {
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			if (item?.data?.action === 'unbanUser') {
+				const form = document.getElementById(`${portletNamespace}fm`);
+
+				if (!form) {
+					return;
+				}
+
+				postForm(form, {
+					data: {
+						cmd: 'unban',
+					},
+					url: banUsersURL,
+				});
```

Comment:
See https://github.com/liferay-frontend/liferay-portal/pull/759#discussion_r574501571

If it's just one line in `if` block, `postForm`, it's hard to justify an early `return`.